packages/suite/src/components/suite/modals/ReduxModal/TransactionReviewModal/TransactionReviewModalContent.tsx

```diff
@@ -71,9 +71,10 @@ export const TransactionReviewModalContent = ({
             : selectSendFormDraftByAccountKey(state, account?.key),
     );
 
-    const decreaseOutputId = precomposedTx?.useNativeRbf
-        ? precomposedForm?.setMaxOutputId
-        : undefined;
+    const isRbfAction = precomposedTx !== undefined && isRbfTransaction(precomposedTx);
+
+    const decreaseOutputId =
+        isRbfAction && precomposedTx?.useNativeRbf ? precomposedForm?.setMaxOutputId : undefined;
```

Comment:
nit: useless optional chaining
```suggestion
        isRbfAction && precomposedTx.useNativeRbf ? precomposedForm?.setMaxOutputId : undefined;
```

Reply:
1aabb1b7515a434e01e2e48366db9635d2f3edf2